pathfinding_service.rst

```diff
@@ -355,6 +356,18 @@ existence. A Pathfinding service SHOULD accept the message if and only if the se
 address recovered from the signature.
 
 
+Routing feedback
+================
+
+The PFS is interested in receiving feedback about the routes it proposed to Raiden clients. For that reason the routing feedback mechanism is used.
+
+When a client requests a route from a PFS (see :ref:`pfs_api_paths`), the PFS also returns a *feedback token*. This token is a UUID in version 4.
```

Comment:
`PFS also returns a *feedback token*` maybe say: `PFS returns a *feedback token* together with the number of routes requested`